### ISSUE TITLE
License

### ISSUE THREAD

**Issue by allprogrammers** [waqarsaleem/Sudoku#1]
I think a license should be added to allow coders to clone, experiment and extend the code.

**Comment by waqarsaleem** [comment]
Agreed. I do not know much about licenses. Please make a PR to add one and I will incorporate it.
